src/i_oalsound.c

```diff
@@ -546,6 +546,22 @@ static boolean IsPaddedSound(const byte *data, int size)
     return true;
 }
 
+static void FadeInMono8(byte *data, ALsizei size, ALsizei freq)
+{
+    const int fadelen = freq / 1000; // 1 ms
```

Comment:
Could you add a const variable or macro to set the fade-in time in ms at compile-time to easily try other values than `1 ms` for all the sound formats?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1220/commits/b7d9a7bb8f1aada8fa9c9ac5eaa737ffa64f9252.

Reply:
There doesn't seem to be any harm in halving the value (0.5 ms or 500 us), but I haven't tested it as much as 1 ms.

Reply:
After more testing, 1 ms seems like a good value for a wide range of sound files.

I haven't merged this PR yet because I've been reading various DSP topics. There are other solutions that may be worth pursuing so I want to look at them closer. In general, this PR is simple, works, and is fast. The alternatives seem slower and require additional libraries but allow for more control over the sound quality and don't require a fade-in (or reduce it to an even shorter duration).

Reply:
They're is still the issue of 8- bit sound files loaded through libsndfile, or am I wrong?

Reply:
> They're is still the issue of 8- bit sound files loaded through libsndfile, or am I wrong?

I think that's for a separate PR.

Reply:
Alright then.